rasterio/rio/clip.py

```diff
@@ -99,16 +100,23 @@ def clip(ctx, files, output, bounds, like, driver, projection,
             else:
                 raise click.UsageError('--bounds or --like required')
 
-            window = src.window(*bounds)
+            bounds_window = src.window(*bounds)
+
+            # Call int_reshape to get the window with integer height
+            # and width that contains the bounds window.
+            out_window = int_reshape(bounds_window)
```

Comment:
seems like this could be simplified to:

`out_window = int_reshape(src.window(*bounds))`